tests/wicked/startandstop_sut.pm

```diff
@@ -109,6 +109,39 @@ sub run {
     }
     mutex_create("test_5_ready");
 
+    $self->before_scenario('Test 7', 'Bridge - ifdown, create new config, ifreload, ifdown, ifup', $iface);
+    $config = '/etc/sysconfig/network/ifcfg-br0';
+    $self->get_from_data('wicked/ifcfg/br0',    $config);
+    $self->get_from_data('wicked/ifcfg/dummy0', $dummy);
+    $self->setup_bridge($config, $dummy, 'ifup');
+    assert_script_run('wicked ifdown --timeout infinite br0');
+    assert_script_run('wicked ifdown --timeout infinite dummy0');
+    $results{7} = sub {
+        $res = script_run(q(ip link | grep 'dummy0\|br0'));
+        return 'FAILED' if ($res == 0);
+
+        assert_script_run('wicked ifreload --timeout infinite all');
```

Comment:
<thinking loud mode>with this approach you basically can drop usage of `assert_script_run`  at all ? just return failed state everywhere</thinking loud mode>

Reply:
Yes for this approach we could do with `script_run()`. 
But it depends on how we plan to handle the errors. I think the idea of refactoring is to run each test step in a single test_module and then It would make sense to use the common error handling with `assert_script_run()` and cleanup in `post_fail_hook`.